content/en/serverless/azure_app_services/azure_app_services_linux.md

```diff
@@ -118,3 +181,5 @@ Share the content of the **Log stream** with [Datadog Support][14].
 [10]: https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent&code-lang=php
 [11]: https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent&code-lang=python
```

Comment:
For all runtimes, can we link to the client instructions for Dogstatsd directly, e.g. https://docs.datadoghq.com/developers/dogstatsd/?tab=java&code-lang=java#install-the-dogstatsd-client. The rest of the page is not super relevant and somewhat confusing for a first time customer.